readme.md

```diff
@@ -20,7 +20,7 @@ And access the application at http://localhost:4000/
 Member data is located under `_collections/members`. Copy an existing profile, name it something unique (preferably using your username on the devanooga Slack) and fill it out using these properties:
 
 - **`name`** (required) Your full name as it should appear on the site.
-- **`gravatar`** (required) Your gravatar hash. [Use this tool to get your gravatar hash.](http://aruljohn.com/gravatar/) _For example, @StrangeWill’s result is `http://www.gravatar.com/avatar/6789f8bd72612e941f9a7ae6f414b2ea?s=48&d=identicon`; the hash is this part: `6789f8bd72612e941f9a7ae6f414b2ea`._
+- **`gravatar`** (required) Your gravatar hash. [Use this tool to get your gravatar hash](https://en.gravatar.com/site/check/), after you enter your email address it will be displayed as `Email Hash`. _For example, @StrangeWill’s result is `Email Hash: 6789f8bd72612e941f9a7ae6f414b2ea`._
```

Comment:
I guess a quick blurb about it is fine. I'd probably just say something like "this is what we use to display your picture on our page, click here for more details" or something, with a link to something on gravatar's site.